### PR TITLE
MAM-4032-listening-to-audio-not-working

### PR DESCRIPTION
**Pull request by derspyy**
part of #365, also.

### CODE REVIEW

File: Mammoth/Views/Cells/PostCardCell/PostCardVideo.swift [hunk 1]
```diff
@@ -450,9 +450,11 @@ final class PostCardVideo: UIView {
             guard self.media != media else { return }
             
             self.media = media
-            if let videoURL = URL(string: media.remoteURL ?? media.url), let previewImageURL = URL(string: media.previewURL ?? "") {
+            if let videoURL = URL(string: media.remoteURL ?? media.url) {
                 
                 loadingIndicator.startAnimating()
+                
+                let previewImageURL = URL(string: media.previewURL ?? "")
```
Comment (bnolens) [inline Mammoth/Views/Cells/PostCardCell/PostCardVideo.swift:457]: I'd say, unwrapping the optional like in this example would be cleaner
```
if let previewImageURL = URL(string: media.previewURL) {
     previewImage.sd_setImage(with: previewImageURL)
}

**Review by bnolens — CHANGES_REQUESTED** [review]
Please have a look at my comment

File: Mammoth/Views/Cells/PostCardCell/PostCardVideo.swift (hunk 1 above)
Comment (derspyy) [reply Mammoth/Views/Cells/PostCardCell/PostCardVideo.swift:457]: you're right.

Comment (derspyy) [reply Mammoth/Views/Cells/PostCardCell/PostCardVideo.swift:457]: well, actually, 

```
if let previewImageURL = media.previewURL{
     previewImage.sd_setImage(with: URL(previewImageURL))
}
```

Comment (bnolens) [reply Mammoth/Views/Cells/PostCardCell/PostCardVideo.swift:457]: Yes that works too

**Review by bnolens — APPROVED** [review]
LGTM!

**Pull request merged**